modules/core/src/test/resources/org.apache.ignite.util/GridCommandHandlerClusterByClassTest_help.output

```diff
@@ -63,10 +63,10 @@ This utility can do the following commands:
     control.(sh|bat) --diagnostic
 
   Enable rolling upgrade:
-    control.(sh|bat) --rolling-upgrade on
+    control.(sh|bat) --rolling-upgrade on [--force] [--yes]
```

Comment:
Looks like it should be updated in accordance with new commands: start/finish/force
Please check `GridCommandHandlerClusterByClassTest.testHelp`